Add emojis to command output, edit UG (#169)
AY2122S1-CS2103T-T09-1/tp: docs/UserGuide.md

1. Refer to the [Features](#features) section for more commands and features in StudyTracker.

<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
Check out **<a href="#command-structure">How to read Commands in StudyTracker</a>** section if you are unfamiliar 
with CLI syntax!

> </div>

Author: Sorry idk why I changed this line

* `Name` — name of study spot
* `Tags` — user-defined tags for the study spot
* `Address` — address of study spot
* `Operating Hours` — operating hours of study spot on weekdays and weekends respectively

> * `Favourite` — the presence of this icon shows that the study spot is added to favourites, and vice versa

Reviewer: ```suggestion
* `Favourite` — this icon indicates the study spot is a favourite
```
Would this be more straightforward?

| Input     | Output | Usage of Alias | Output |
| ----------- | ----------- | ---------- | ------------- |

> | `alias al/MyList cmd/list` | Added alias myList: [list] | `myList` | `Listed all study spots` |

Reviewer: * Is there a reason why `Added alias myList: [list]` should not be in codeblocks?
* `myList` should be `MyList` in the output column
* this example may not be the best as we wrote that `alias` command helps to shorten commands but it lengthened the command input in this example...
* Perhaps we should include that alias is case-sensitive?

Author: 1. Isn't Added alias myList: [list] an output and not a command? I think for the UG all the outputs have no codeblocks. 
2. okay!
3. changed MyList to ml
4. okay!
